src/download.ts

```diff
@@ -91,20 +105,53 @@ export async function downloadPage(title: string, link: string): Promise<void> {
         node.childNodes[0].childNodes[0].childNodes[1].appendChild(content);
         node.childNodes[0].childNodes[0].childNodes[0].remove();
       }
+
+      return languages;
     }, { SAVE_AS, SAVE_LESSON_AS });
 
-    if (SAVE_AS === SAVE_LESSON_AS.PDF) {
-      await page.emulateMediaType('screen');
-      await page.pdf({
-        path: `${SAVE_DESTINATION}/${normalizedTitle}.pdf`,
-        printBackground: true,
-        format: 'A4',
-        margin: { top: 0, right: 0, bottom: 0, left: 0, }
-      });
+    /**
+     * If lesson has multiple language and user set multiLanguage to true then download all language.
+     */
+    if (languages.length > 0 && MULTI_LANGUAGE) {
```

Comment:
This should be 1 as if there is only one language available we don't need to put it in separate folder.